time-elements.js

```diff
@@ -561,7 +563,9 @@
     var options = {
       hour: el.getAttribute('hour'),
       minute: el.getAttribute('minute'),
-      second: el.getAttribute('second')
+      second: el.getAttribute('second'),
+      timeZone: el.getAttribute('timeZone'),
+      timeZoneName: el.getAttribute('timeZoneName')
```

Comment:
But, if we wanted to force the timezone used, we could just have the server render the timestamp in the specified timezone itself. We wouldn't need JavaScript for that. The purpose of this project is to take a machine-readable timestamp rendered from the server and transform it into a local time based on the user's browser.